lib/transformer/tests/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "near-bindgen",
```

Comment:
"near-bindgen"?

Reply:
Oops! Yeah it was a test originally used there.